morango/models.py

```diff
@@ -421,15 +422,16 @@ def serialize(self):
             # case if model is morango mptt
             if f.attname in getattr(self, '_internal_mptt_fields_not_to_serialize', '_internal_fields_not_to_serialize'):
                 continue
+            if f.attname in data:
+                continue
             data[f.attname] = f.value_from_object(self)
         return data
 
     @classmethod
-    def deserialize(cls, dict_model):
+    def deserialize(cls, dict_model, **kwargs):
```

Comment:
When do you project we would need to pass in `**kwargs`?

Reply:
from the parent class? atleast thats how i did it in https://github.com/learningequality/kolibri/pull/2496/files#diff-6efacfe268cfdb28150182ee233780aeR509
wasn't sure what the best pattern was to do this.

Reply:
https://github.com/learningequality/kolibri/issues/2817